ee/vellum_ee/workflows/display/nodes/vellum/error_node.py

```diff
@@ -11,6 +11,8 @@
 
 class BaseErrorNodeDisplay(BaseNodeVellumDisplay[_ErrorNodeType], Generic[_ErrorNodeType]):
     error_output_id: ClassVar[Optional[UUID]] = None
+    error_inputs_by_name: ClassVar[Dict[str, Any]] = {}
+    name: ClassVar[str] = "error-node"
```

Comment:
Oh wow, didn't realize error nodes have names.

Reply:
They don't...

Reply:
They actually do in the main repo for some reason. Check out `ErrorNodeData`.